src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/ubuntu/14.04/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -4,21 +4,21 @@
   <PropertyGroup>
     <SkipPackageFileCheck>true</SkipPackageFileCheck>
     <PackageTargetRuntime>ubuntu.14.04-$(PackagePlatform)</PackageTargetRuntime>
-    <!-- only build for x64 -->
-    <PackagePlatforms>x64;</PackagePlatforms>
+    <!-- build for x64, arm -->
+    <PackagePlatforms>x64;arm;</PackagePlatforms>
   </PropertyGroup>
   <ItemGroup>
     <NativeSplittableBinary Include="$(BinDir)libcoreclr.so" />
-    <NativeSplittableBinary Include="$(BinDir)libcoreclrtraceptprovider.so" />
+    <NativeSplittableBinary Condition="'$(PackagePlatform)' != 'arm'" Include="$(BinDir)libcoreclrtraceptprovider.so" />
```

Comment:
I am building on the Raspberry PI 3 Raspian. 
So, when the build is properly enabled (there was one more place where the feature was set for PAL, I was setting it in clrdefinitions.cmake only before), it fails in `/usr/include/urcu/uatomic/generic.h`, which is a header that belongs to the lttng. I can see it is a known issue that was reported and fixed in lttng more than a year ago (http://comments.gmane.org/gmane.linux.kernel.tracing.lttng.devel/10012). So maybe there are too old lttng libraries on my Raspbian or something. But I have just installed the `liblttng-ust-dev` package avaiable in Raspbian. And the version and header seems to be the same as the one on my AMD64 ubuntu, so I wonder if the problem is in clang 3.7 being more sensitive.